sdk/storage/azure-storage-queue/azure/storage/queue/models.py

```diff
@@ -298,44 +295,40 @@ class QueuePropertiesPaged(Paged):
         begin with the specified prefix.
     :param int results_per_page: The maximum number of queue names to retrieve per
         call.
-    :param str marker: An opaque continuation token.
+    :param str continuation_token: An opaque continuation token.
     """
-    def __init__(self, command, prefix=None, results_per_page=None, marker=None):
-        super(QueuePropertiesPaged, self).__init__(command, None)
+    def __init__(self, command, prefix=None, results_per_page=None, continuation_token=None):
+        super(QueuePropertiesPaged, self).__init__(
+            self._get_next_cb,
+            self._extract_data_cb,
+        )
```

Comment:
The other models pass `continuation_token` into the `super.__init__`?

Reply:
Yes, that's better (in this case it was not broken because I was setting self.continuation_token latter, but that's better to pass it in the init).
Fixed